web/src/lib/auth/sessionHelper.ts

```diff
@@ -108,16 +109,21 @@ export const getActiveUser = async (): Promise<ActiveUser> => {
       "custom:identityId": credentials.identityId,
     });
   }
-  return cognitoPayloadToActiveUser(cognitoPayload);
+  const encounteredMyNjLinkingError = AccountLinkingErrorStorageFactory().getEncounteredMyNjLinkingError();
+  return cognitoPayloadToActiveUser(cognitoPayload, { encounteredMyNjLinkingError });
 };
 
-const cognitoPayloadToActiveUser = (cognitoPayload: CognitoIdPayload): ActiveUser => {
+const cognitoPayloadToActiveUser = (
+  cognitoPayload: CognitoIdPayload,
+  params: { encounteredMyNjLinkingError: boolean | undefined }
+): ActiveUser => {
```

Comment:
[sand] I said this somewhere else, too. Should all of these positional params be included in the new params object? It feels weird to me to have competing styles here.